src/OrbitBase/FileTest.cpp

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+﻿// Copyright (c) 2021 The Orbit Authors. All rights reserved.
```

Comment:
Just out of curiosity.. do you know whats going on here? Did the encoding of the file change? Is there a hidden unicode character here?

Reply:
Yep that's it, I added "UnicodeFileName🚀.txt" below, that changed the file encoding because of the Unicode character.